types/dialogify-browser/index.d.ts

```diff
@@ -0,0 +1,228 @@
+// Type definitions for non-npm package Dialogify 1.0
+// Project: https://github.com/OneupNetwork/dialogify
+// Definitions by: moontai0724 <https://github.com/moontai0724>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.4
```

Comment:
why this requirement?

Reply:
Cause I run `npm test dialogify-browser` locally and dtslint told me I need to add this requirement:

```
/home/ubuntu/DefinitelyTyped/types/dialogify-browser/index.d.ts:148:21
ERROR: 148:21  expect  Compile error in typescript@4.3 but not in typescript@4.4.
Fix with a comment '// Minimum TypeScript Version: 4.4' just under the header.
An index signature parameter type cannot be a union type. Consider using a mapped object type instead.

    at testTypesVersion (/home/ubuntu/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:194:15)
    at async runTests (/home/ubuntu/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:170:13)
    at async main (/home/ubuntu/DefinitelyTyped/node_modules/@definitelytyped/dtslint/dist/index.js:79:9)
```

Reply:
what is `dialogify-browser` relation to this `dialogify`?